Resources/doc/annotations/routing.rst

```diff
@@ -1,190 +1,61 @@
 @Route and @Method
 ==================
 
-.. caution::
+**Routing annotations of the SensioFrameworkExtraBundle are deprecated** since
+version 5.2 because they are now a core feature of Symfony.
 
-    Routing annotations of the SensioFrameworkExtraBundle are deprecated since version 5.2.
-    Configuring routes with annotations is now a core feature of Symfony. Please use it
-    instead as explained in :doc:`/routing`.
+How to Update your Applications
+-------------------------------
 
-Usage
------
+``@Route`` Annotation
+~~~~~~~~~~~~~~~~~~~~~
 
-The ``@Route`` annotation maps a route pattern with a controller::
+The Symfony annotation has the same options as the SensioFrameworkExtraBundle
+annotation, so you only have to update the annotation class namespace:
 
-    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+.. code-block:: diff
 
-    class PostController extends Controller
+    -use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+    +use Symfony\Component\Routing\Annotation\Route;
+
+    class DefaultController extends Controller
     {
         /**
          * @Route("/")
          */
-        public function indexAction()
+        public function index()
         {
             // ...
         }
     }
 
-The ``index`` action of the ``Post`` controller is now mapped to the ``/``
-URL. This is equivalent to the following YAML configuration:
+``@Method`` Annotation
+~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: yaml
+The ``@Method`` annotation from SensioFrameworkExtraBundle has been removed.
+Instead, the Symfony ``@Route`` annotation defines a new ``methods`` option to
```

Comment:
The `methods` option was introduced in Symfony 2.2. So I would not call it "new" here.

Reply:
see symfony/symfony#6738 for the PR adding it in Symfony